pkgs/development/python-modules/traittypes/default.nix

```diff
@@ -14,47 +13,40 @@
 
 buildPythonPackage rec {
   pname = "traittypes";
-  version = "unstable-2019-06-23";
-  format = "setuptools";
+  version = "unstable-2020-07-17";
+  pyproject = true;
 
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "jupyter-widgets";
     repo = pname;
-    rev = "0a030b928991dec732c17a7a1cb13acbcd7650a2";
-    sha256 = "0rlm5krmq6n8yi47dgdsjyrkz3m079pndpbzkz2gx98pb3jd9pjs";
+    rev = "af2ebeec9e58b73a12d4cf841bd506d6eadb8868";
+    hash = "sha256-q7kt8b+yDHsWML/wCeND9PrZMVjemhzG7Ih1OtHbnTw=";
   };
 
-  patches = [
-    (fetchpatch {
-      name = "fix-intarray-test.patch";
-      url = "https://github.com/minrk/traittypes/commit/a02441e5b259e5858453a853207260c9bd4efbb5.patch";
-      sha256 = "120dsvr5nksizw75z1ah3h38mi399fxbvz5anakica557jahi0aw";
-    })
-  ];
+  postPatch = ''
+    substituteInPlace traittypes/tests/test_traittypes.py \
+      --replace-fail "np.int" "int"
+  '';
```

Comment:
I guess upstream is dead, but it might make sense to send this as a PR for anyone else packaging this?

Reply:
I felt this was the simplest solution, and I don't really have the energy to send patches to every single upstream that isn't maintained but needs patches. If someone else wants to, they're welcome to, and I'd be glad to adopt that patch here instead.